backend/src/akvo/lumen/component/handler.clj

```diff
@@ -2,6 +2,8 @@
   (:require [compojure.core :as compojure.core]
             [compojure.response :as compojure.res]
             [integrant.core :as ig]
+            [akvo.lumen.specs.components :refer (integrant-key)]
```

Comment:
Since we used vectors most of the places, is there a reason for using the list notation?